### PR TITLE
placeholder+i18n support, addresses #9 and #10

### PR DESCRIPTION
**Pull request by JSey**
added some localizations for the submit button, the composer placeholder and the tooltip for the bullhorn icon. Languages: English+German

### CODE REVIEW

File: assets/javascripts/discourse/templates/components/babble-composer.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <div class="babble-composer">
-  {{textarea value=text}}
-  <button {{action 'submit'}} {{bind-attr disabled="submitDisabled"}} class="btn btn-primary pull-right">Send</button>
+  {{textarea value=text placeholder=i18nPlaceholder }}
```
Comment (gdpelican) [inline assets/javascripts/discourse/templates/components/babble-composer.hbs]: I think you can just do the following here, without using a component function?
```
title="{{i18n 'babble.placeholder'}}"
```

Like here:
https://github.com/discourse/discourse/blob/master/app/assets/javascripts/admin/templates/customize_colors.hbs#L63

Comment (JSey) [inline assets/javascripts/discourse/templates/components/babble-composer.hbs]: Oh, I didn't know you can nest those like that - nice! Will do - after my vacation. Thx!

**Comment by gdpelican** [comment]
Hm, as far as I can tell, this should be working perfectly, but I can't seem to get the translations to load correctly into the client. (Running `i18n.translations` in the browser gives me an object with all the translations in it, and the babble ones aren't there..)

Not sure what the missing special sauce is, as far as I can tell, we're doing the same thing as, say, here: https://github.com/discourse/discourse-solved

**Comment by gdpelican** [comment]
Closing this in favor of #19 (I've kept your commits in there, but want to be able to make a quick change before merging it), thanks @JSey!

**Pull request closed without merging**